pkg/generators/builder/builder.go

```diff
@@ -173,7 +195,13 @@ func (b *BuilderPatternGenerator) Namers(c *generator.Context) namer.NameSystems
 }
 
 func (b *BuilderPatternGenerator) Imports(c *generator.Context) (imports []string) {
-	importLines := []string{}
+	b.imports.AddType(&types.Type{
+		Name: types.Name{
+			Name:    "Runtime",
+			Package: "k8s.io/apimachinery/pkg/util/runtime",
+		},
+	})
+	importLines := []string{"k8s.io/apimachinery/pkg/runtime"}
```

Comment:
just curious, shouldn't this importLine already be in b.imports.ImportLines() after `AddType`?
I would've thought so since other imports are not explicitly added